src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -9,6 +9,12 @@
 >> Abs(2)
 2
 
+>> Abs(2147483649) // testing a number larger than the interger range 
+2147483649
+
+>> Abs(9007199254740991) // testing a number larger than the float range (-9007199254740991 to 9007199254740991, (+/- 2^53 - 1))
```

Comment:
Suggestion: `// testing the largest consecutive integer that represented without loss in a floating point`. "larger than the float range" isn't correct, since numbers larger than that can be represented as well.

Reply:
Same for the test with negative number

Reply:
Okay, thank you!